README.md

```diff
@@ -102,6 +102,28 @@ make uninstall
 ```
 This will delete all custom resource definitions created for the controller, as well as the `che-workspace-controller` namespace.
 
+### Deploying the controller with olm
```

Comment:
When I deploy using `make olm_full_start`, the `Component` CRD is not listed in the provided APIs:

![Screenshot from 2020-05-25 16-00-25](https://user-images.githubusercontent.com/16168279/82840522-ea3c5580-9ea0-11ea-9a6f-38dfbd48d5b3.png)

is this intentional?

Reply:
I'm not sure but it might be related to the fact that: https://docs.openshift.com/container-platform/4.3/operators/operator_sdk/osdk-generating-csvs.html#osdk-hiding-internal-objects_osdk-generating-csvs is set in the csv. Though technically Workspace routing should also be hidden in this case

Reply:
Ah I figured it out. There is a bug `make olm_start` where the operatorsource.yaml wasn't changing registryNamespace to the correct one so it was still trying to use the old crd registry